src/stylesheets/components/_phase-banner.scss

```diff
@@ -1,14 +1,3 @@
-.app-phase-banner {
```

Comment:
Should we also remove where this class is being set https://github.com/alphagov/govuk-design-system/blob/main/views/partials/_banner.njk#L20

Reply:
Good shout – have pushed a commit to do just that.